### ISSUE TITLE
[BUG] Spawned Node3D sometimes revert to earlier setting

### ISSUE THREAD

**Issue by aXu-AP** [dreadpon/godot_spatial_gardener#44]
**Godot version**
v4.2.2.stable.official [15073afe3]

**Plugin version**
1.3.2

**Platform**
Windows 10

**Issue description**
In my scene I have set up a few types of bushes and trees. For a bush I initially set up collision from another tree as Spawned Node3D. Later I changed it to another scene that fits more to the shape. However, sometimes it randomly reverts to the scene I initially set it to. I change it back again and for some time it works, but then again reverts.

**Steps to Reproduce**
Unfortunately I haven't been able to notice any rhythm or reason as to how this happens...

**Minimal reproduction project**
See above. This might be one of those hard to catch bugs since it happens very sporadically, and for some reason on one plant at that.

**Comment by dreadpon** [comment]
Have a few ideas where this may be coming from: there is an array that is shared among all octree nodes, containing the list of meshes and scene to be used for different LODs. Probably something goes wrong when updating this array (possibly related to how Godot caches resources?).
Will investigate, but not urgently.

**Comment by aXu-AP** [comment]
Yeah I think it's not urgent as it doesn't happen not too often and it might be hard to track down. I'll post more info if I notice something new.

**Comment by dreadpon** [comment]
This is likely fixed by https://github.com/dreadpon/godot_spatial_gardener/commit/a5802ba41cc6e47604c6e25c5e18983de7a7934f, https://github.com/dreadpon/godot_spatial_gardener/commit/a5802ba41cc6e47604c6e25c5e18983de7a7934f since these changes overhaul the instance spawning process a lot and should make it more reliable.
